test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -225,7 +226,42 @@ const appLinkWithoutLaunchMock = RequestMock()
   .onRequestTo(customAppLink)
   .respond('<html><h1>open app link</h1></html>');
 
-fixture('Device Challenge Polling View with the Loopback Server, Custom URI and Universal Link approaches');
+const username = 'john.smith@okta.com';
```

Comment:
you can use following to share `variables` between the tests

https://testcafe.io/documentation/402708/reference/test-api/testcontroller/ctx

Reply:
We do not need to share username between tests. Const variable is fine.

Reply:
I meant to say instead of creating these variable globally. we should use `testContext`

Reply:
@pradeepdewda-okta We can access the ctx from within the test. However we still need to use it in the RequestMock constant to define request url path (eg, In line #238 and #246 in this file, username is part of URL for UL/AppLink. We need to mock the request/response). There is no ctx when defining RequestMock

Reply:
okay.. i was just focusing on `await t.expect(attributes.src).contains('login_hint='+encodeURIComponent(username));`